### ISSUE TITLE
Length of downloaded audio not saved

### ISSUE THREAD

**Issue by prashantchawla** [TeamNewPipe/NewPipe#274]
I think the length of downloaded audio files is not being saved with the file. As a result, many audio players just show the length as "0:00" or "Buffering..." and seeking the audio forward/backward is not possible. Can this be fixed?
![screenshot_2016-04-19-14-03-16_naman14 timber](https://cloud.githubusercontent.com/assets/10388652/14632833/47a81868-0638-11e6-97e8-f6e2492e6525.png)

**Comment by theScrabi** [comment]
Well we'd have to change the audiofiles, and for me that is low priorety. Means, if someone wants to do that he can do so. I can't at the moment.

**Comment by theScrabi** [comment]
I can't confirm. Timber always shows the length, and makes seeking possible.

**Comment by prashantchawla** [comment]
Sorry for late update. Had my exams going on. 
So I tried to narrow down the issue and this is what I observed: I'm on MIUI 7.2 (Android 5.1) and I'm having this issue with all the players (MIUI Player, Timber and Apollo). However, on playing the same file on a Kitkat phone or my Windows PC, the issue is not observed. Also, MIUI is able to properly play m4a files from other youtube downloaders. I'm not sure if this issue is specific to MIUI or common to all 5.1 devices. Hope this will help to pinpoint the issue.
